src/core/cgroup.h

```diff
@@ -3,6 +3,9 @@
 
 #include <stdbool.h>
 
+#if HAVE_LIBBPF
+#include "bpf-lsm.h"
```

Comment:
hmm, this header can be imported always, no? i.e. it's written to also be clean to import if HAVE_LIBBPF is 0, no?

I'd generally try to minimize ifdeffery in consuming files, and place it in implementing files instead. Thus, i'd just include bpf-lsm.h here unconditionally without ifdeffery